apis/posts/views.py

```diff
@@ -1,10 +1,12 @@
 # Using serializers and Rest framework:
```

Comment:
I don't think this file contains any valuable changes so we could reset it all together?